weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -28,9 +28,20 @@
 import json
 import logging
 import threading
-import typing
 from collections import Counter, defaultdict
 from contextlib import contextmanager
+from typing import (
+    Any,
+    Dict,
+    Iterator,
+    List,
```

Comment:
Since we require Python 3.9+ I believe we can just use e.g. `list` instead of `typing.List`

Reply:
Fixing